### PR TITLE
Output format changes for aws-cloudformation-package command

### PR DESCRIPTION
**Pull request by DiegoPomares**
Use JSON as default format for output of "aws cloudformation package" command, added an flag for YAML.

### CODE REVIEW

**Comment by sanathkr** [comment]
You should default to YAML and provide a use-json override. YAML dump was the old behavior and you should retain it for the majority of customers that are happy with it

**Comment by sanathkr** [comment]
Also need unit tests

**Comment by DiegoPomares** [comment]
YAML is the default output format now. Updated a unit test to get coverage back to 100%.

File: awscli/customizations/cloudformation/yamlhelper.py [hunk 1]
```diff
@@ -53,12 +53,18 @@ def intrinsics_multi_constructor(loader, tag_prefix, node):
     return {cfntag: value}
 
 
+def str_representer(dumper, node):
```
Comment (sanathkr) [inline awscli/customizations/cloudformation/yamlhelper.py]: We don't need this, do we? customers having trouble with YAML integers can use JSON. I don't want to uglify YAML with quotes everywhere

**Comment by sanathkr** [comment]
Looks good. Thanks for doing.

**Comment by sanathkr** [comment]
@jamesls @kyleknap Want to review as well?

**Comment by dstufft** [comment]
Looks good, thanks!

**Pull request merged**